### PR TITLE
#165413103 reset password via email

### PR DESCRIPTION
**Pull request by Kabalisa**
#### Description
Resetting password is a very important feature as forgetting is the nature of humans. this feature facilitates authorshaven's users to reset their user's account password using their email.
#### Type of change
N/A
#### How Has This Been Tested?
N/A
#### Checklist:
N/A
#### Pivotal Tracker ID
[#165413103](https://www.pivotaltracker.com/story/show/165413103)

### CODE REVIEW

**Review by Niyitangasam — APPROVED** [review]

File: config/swagger.js [hunk 1]
```diff
@@ -10,8 +10,8 @@ const swaggerDefinition = {
     version: '1.0.0',
     description: 'A Social platform for the creative at heart'
   },
-  host: `${process.env.BASE_URL}`,
-  basePath: '/api/v1'
+  host: 'localhost:5000',
+  basePath: '/api'
```
Comment (macphilips) [inline config/swagger.js]: Change back to `api/v1`

File: config/swagger.js [hunk 2]
```diff
@@ -10,8 +10,8 @@ const swaggerDefinition = {
     version: '1.0.0',
     description: 'A Social platform for the creative at heart'
   },
-  host: `${process.env.BASE_URL}`,
-  basePath: '/api/v1'
+  host: 'localhost:5000',
```
Comment (macphilips) [inline config/swagger.js]:
```suggestion
  host: `${process.env.BASE_URL}`,
```

File: helpers/index.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+/* eslint-disable require-jsdoc */
```
Comment (macphilips) [inline helpers/index.js]: Remove this line, and add the js-doc it is important

File: helpers/utils/mail-sender.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    const sendEmailPromise = new Promise((resolve, reject) => {
+      const email = {
+        from: process.env.APP_EMAIL,
+        to: infos.email,
+        subject: infos.subject,
+        html: infos.html
+      };
+
+      client.sendMail(email, (err, info) => {
+        // console.log(err, info);
```
Comment (macphilips) [inline helpers/utils/mail-sender.js]: remove `console.log`

**Review by jabichris — COMMENTED** [review]
Can you resolve that issue with  travis CI

**Comment by Kabalisa** [comment]
> Can you resolve that issue with Travis CI

@jabichris the Travis ci is working that id the history of Travis builds which includes previous builds which were not running

**Review by jabichris — APPROVED** [review]

**Pull request merged**